Proton/Sources/Swift/Editor/EditorView.swift

```diff
@@ -108,13 +108,45 @@ open class EditorView: UIView {
     let richTextView: RichTextView
     let context: RichTextViewContext
     var needsAsyncTextResolution = false
+    
+    public var contentSize: CGSize {
+        richTextView.alwaysBounceVertical = true
```

Comment:
It is not a good idea to have side-effect i.e change a property when accessing another property.